dev-ruby/with_advisory_lock/with_advisory_lock-5.1.0.ebuild

```diff
@@ -27,6 +27,8 @@ ruby_add_bdepend "test? (
 	>=dev-ruby/activerecord-6.1:*[sqlite]
 	dev-ruby/maxitest
 	dev-ruby/mocha
+	dev-ruby/sqlite3:0
```

Comment:
The correct version of sqlite should already be pulled in through the sqlite USE flag for activerecord, although I seem to recall that I should check the dependencies for that in the older versions.